app/test/prepare_proposal_test.go

```diff
@@ -137,6 +137,22 @@ func TestPrepareProposalFiltering(t *testing.T) {
 	require.NoError(t, err)
 	noAccountTx := []byte(testutil.SendTxWithManualSequence(t, encConf.TxConfig, kr, nilAccount, accounts[0], 1000, "", 0, 6))
 
+	// create a tx that can't be included in a 64 x 64 when accounting for the
+	// pfb along with the shares
+	tooManyShareBtx := blobfactory.ManyMultiBlobTx(
+		t,
+		encConf.TxConfig,
+		kr,
+		testutil.ChainID,
+		accounts[3:4],
+		infos[3:4],
+		blobfactory.NestedBlobs(
+			t,
+			testfactory.RandomBlobNamespaces(tmrand.NewRand(), 4000),
+			[][]int{repeat(4000, 1)},
+		),
```

Comment:
[question] out of curiosity how many PFB shares is this? It seems like 4000 blobs shares + x PFB shares. 

The PFB shares need to be >= 96 in order for the total number of shares to be >= 4096.

Reply:
good point, the back of the napkin reasoning here is each blob has a size, namespace, and blob share commitment, so the overhead is atleast ~72 bytes + encoding overhead. So by this math we're well over the <50KB found in 96 shares